pkg/crc/machine/start.go

```diff
@@ -541,7 +541,7 @@ func configProxyForCluster(ocConfig oc.Config, sshRunner *crcssh.Runner, sd *sys
 
 func waitForProxyPropagation(ocConfig oc.Config, proxyConfig *network.ProxyConfig) {
 	checkProxySettingsForOperator := func() error {
-		proxySet, err := cluster.CheckProxySettingsForOperator(ocConfig, proxyConfig, "redhat-operators", "openshift-marketplace")
+		proxySet, err := cluster.CheckProxySettingsForOperator(ocConfig, proxyConfig, "marketplace-operator", "openshift-marketplace")
```

Comment:
Is it changed recently, can you also add refer doc to commit message?

Reply:
Apparently this deployment was always named like this.
https://github.com/operator-framework/operator-marketplace/blame/e3b79ea32bbaa898b3cc90e3bb2d64907d029ce6/manifests/07_operator.yaml#L4

Maybe redhat-operators was removed?

Reply:
It is not removed, here it is https://github.com/operator-framework/operator-marketplace/tree/master/defaults , I am not sure how it is consumed by operator-marketplace but it is always managed by it, will need to check with latest version of bundle.